diff -rup vim73.orig/src/feature.h vim73/src/feature.h
--- vim73.orig/src/feature.h	2010-07-27 21:45:42.000000000 +0200
+++ vim73/src/feature.h	2010-11-21 05:25:09.532342231 +0100
@@ -936,12 +936,13 @@
  * SYS_VIMRC_FILE	Name of the system-wide .vimrc file.
  */
 /* #define SYS_VIMRC_FILE	"/etc/vimrc" */
-
+#define SYS_VIMRC_FILE "/etc/vimrc"
 /*
  * SYS_GVIMRC_FILE	Name of the system-wide .gvimrc file.
  */
 /* #define SYS_GVIMRC_FILE	"/etc/gvimrc" */
-
+#define SYS_GVIMRC_FILE "/etc/gvimrc"
+	
 /*
  * DFLT_HELPFILE	Name of the help file.
  */
