diff -rup vim73.orig/runtime/doc/Makefile vim73/runtime/doc/Makefile
--- vim73.orig/runtime/doc/Makefile	2010-07-21 20:38:06.000000000 +0200
+++ vim73/runtime/doc/Makefile	2010-11-21 05:19:30.308088771 +0100
@@ -310,6 +310,7 @@ all: tags vim.man evim.man vimdiff.man v
 # Use Vim to generate the tags file.  Can only be used when Vim has been
 # compiled and installed.  Supports multiple languages.
 vimtags: $(DOCS)
+	export VIMRUNTIME=../; \
 	$(VIMEXE) -u NONE -esX -c "helptags ++t ." -c quit
 
 # Use "doctags" to generate the tags file.  Only works for English!
Nur in vim73/runtime/doc: Makefile.orig.
