diff -rup vim73.orig/runtime/syntax/sh.vim vim73/runtime/syntax/sh.vim
--- vim73.orig/runtime/syntax/sh.vim	2010-08-05 22:00:59.000000000 +0200
+++ vim73/runtime/syntax/sh.vim	2010-11-21 05:08:23.026990941 +0100
@@ -42,6 +42,13 @@ if !exists("b:is_kornshell") && !exists(
   endif
 endif
 
+" Neither is_kornshell nor is_bash was explicitly set, and
+" /bin/sh is a bash on SUSE LINUX.
+if exists("b:is_sh")
+  let b:is_bash= 1
+  unlet b:is_sh
+endif
+
 " set up default g:sh_fold_enabled {{{1
 if !exists("g:sh_fold_enabled")
  let g:sh_fold_enabled= 0
