diff -rup vim73.orig/runtime/filetype.vim vim73/runtime/filetype.vim
--- vim73.orig/runtime/filetype.vim	2010-08-03 22:44:00.000000000 +0200
+++ vim73/runtime/filetype.vim	2010-11-21 05:38:50.752088445 +0100
@@ -157,7 +157,7 @@ au BufNewFile,BufRead *.asp
 	\ endif
 
 " Grub (must be before catch *.lst)
-au BufNewFile,BufRead /boot/grub/menu.lst,/boot/grub/grub.conf,/etc/grub.conf	setf grub
+au BufNewFile,BufRead */boot/grub/menu.lst,*/boot/grub/grub.conf,*/etc/grub.conf	setf grub
 
 " Assembly (all kinds)
 " *.lst is not pure assembly, it has two extra columns (address, byte codes)
